### ISSUE TITLE
[Feature Request]: Accessibility

### ISSUE THREAD

**Issue by nsde** [Fabulously-Optimized/vanilla-installer#27]
### Potential Enhancement

TODO:
- [x] Dyslexia font
- [ ] Low blue light-mode (if the OS doesn't support it)
- [ ] Large font option

### How it could be implemented

Settings in menu

**Comment by osfanbuff63** [comment]
It's something to have at some point for sure, but the way the GUI looks right now isn't ideal. I tried playing around with it, but nothing actually seemed to work. As I put in `docs/planned-features.md` I'm wanting to rewrite it at some point, but the current priority is getting a working release.

**Comment by Madis0** [comment]
Does any of this actually need settings? Can't you just ping the system for this?

**Comment by osfanbuff63** [comment]
Good point - but #31 already implements a settings menu, so *in theory* it would be easy to add more options to it.

**Comment by Madis0** [comment]
One does not exclude the other 😄
Where possible, these things should be taken from the system. Where not - an option is fine.

**Comment by osfanbuff63** [comment]
Or both :P

**Comment by osfanbuff63** [comment]
Updates updates updates...

I can't seem to find a way to detect the font set in the OS, but there's even [an official class](https://doc.qt.io/qtforpython/PySide6/QtWidgets/QFontDialog.html) for fonts, so that could be of use.

**Comment by osfanbuff63** [comment]
More updates - decided to implement 1) with a checkbox or similar in the existing settings menu so that I don't need to make a new dialog (although I might do that down the line), 2) doesn't seem like it would be easy to implement, and the OS/monitor should apply it automatically, and I haven't looked into 3) yet.

**Comment by osfanbuff63** [comment]
Because of 1.19.3's release, I'm going to push this back from the v1.0.0 beta release since it's not really needed to get a version out. (Since that beta release is coming **very** soon 👀).

**Comment by osfanbuff63** [comment]
aaah forgot this issue had other things on it

**Comment by Kichura** [comment]
We could implement a similar high contrast theme mode as the one from 1.19.4's resource pack itself although i don't personally know how well that would look.

**Comment by nsde** [comment]
Basically bg-color black and fg-color white, I guess?

**Comment by Kichura** [comment]
morely black because that's what minecraft uses.
